### PR TITLE
TMEDIA-429: Remove arc-site property from promo blocks

### PR DESCRIPTION
**Pull request by nschubach**
## Description
Remove the arc-site variable from being passed in to the promo-blocks.  Fixed propType error.

## Jira Ticket
- [TMEDIA-429](https://arcpublishing.atlassian.net/browse/TMEDIA-429)

## Acceptance Criteria
Remove arcSite from being passed as a query param in feature

## Test Steps
1. Checkout this branch `git checkout TMEDIA-429-remove-arc-site-prop`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/extra-large-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-promo-block,@wpmedia/small-promo-block,@wpmedia/video-player-block,@wpmedia/video-promo-block,@wpmedia/shared-styles`
3. Open `http://localhost/pf/test-all-promos/?_website=the-gazette`
4. Verify that the promos load.

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/shared-styles/_children/promos/extra-large/index.jsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ const ExtraLargePromoPresentation = ({
 
 ExtraLargePromoPresentation.propTypes = {
   content: PropTypes.object,
-  showOverline: PropTypes.string,
+  showOverline: PropTypes.bool,
```
Comment (JackHowa) [inline blocks/shared-styles/_children/promos/extra-large/index.jsx:136]: new? possibly better in another pr?

**Review by JackHowa — COMMENTED** [review]
this looks good. will need to test these, i presume

File: blocks/shared-styles/_children/promos/extra-large/index.jsx (hunk 1 above)
Comment (nschubach) [reply blocks/shared-styles/_children/promos/extra-large/index.jsx:136]: figured it was a small enough change.  Noticed the warning in the console and it's supposed to be a boolean (this is the only one that is not and I think it snuck through in the refactor.)  If you think we need another PR, I can revert that.
```suggestion
  showOverline: PropTypes.string,
```

Comment (JackHowa) [reply blocks/shared-styles/_children/promos/extra-large/index.jsx:136]: no worries it's fine

**Review by JackHowa — APPROVED** [review]
works as expected on target site

**Pull request merged**